### PR TITLE
changed all instances of pitch to hood🦟™️

### PR DESCRIPTION
**Pull request by BrodyKarr**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -158,25 +158,25 @@ public static class constLEDs {
...
-     * The position, in meters, of the center of rotation for the pitch motor
+     * The position, in meters, of the center of rotation for the hood motor
      * relative to the center of the robot (Robot Coordinates).
      * 
      * @see <a href=
      *      "https://docs.wpilib.org/en/stable/docs/software/basic-programming/coordinate-system.html">Robot
      *      Coordinate System</a>
      */
-    public static final Transform3d ROBOT_TO_PITCH = new Transform3d(
+    public static final Transform3d ROBOT_TO_Hood = new Transform3d(
```
Comment (TaylerUva) [inline src/main/java/frc/robot/Constants.java]: Constants should be all caps

**Review by TaylerUva — CHANGES_REQUESTED** [review]
There are now merge conflicts as well

File: src/main/java/frc/robot/RobotPreferences.java [hunk 2]
```diff
@@ -190,47 +190,50 @@ public static final class prefPitch {
      * </p>
      * <b>Units:</b> Degrees per second
      */
-    public static final SN_DoublePreference pitchZeroedVelocity = new SN_DoublePreference("pitchZeroedVelocity", 0.01);
+    public static final SN_DoublePreference hoodZeroedVelocity = new SN_DoublePreference("hoodZeroedVelocity", 0.01);
 
     /**
      * <p>
      * The elapsed time required to consider the pivot motor as zeroed
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/RobotPreferences.java]: PIVOT!!!!!!!! 🚨🚨🚨🚨

File: src/main/java/frc/robot/commands/ManualHoodMovement.java [hunk 3]
```diff
@@ -7,20 +7,21 @@
...
 public class ManualHoodMovement extends Command {
-  Pitch subPitch;
+  Hood subHood;
   DoubleSupplier xAxis;
 
   /** Creates a new ManualHoodMovement. */
-  public ManualHoodMovement(Pitch subPitch, DoubleSupplier xAxis) {
-    this.subPitch = subPitch;
+  public ManualHoodMovement(Hood subhood, DoubleSupplier xAxis) {
+    this.subHood = subhood;
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/commands/ManualHoodMovement.java]: subHood should be uppercase (make the H capitalized)

File: src/main/java/frc/robot/subsystems/Hood.java [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+  }
+
+  public boolean isHoodLocked() {
+    return isHoodAtAngle(desiredLockingHood);
+  }
+
+  @Override
+  public void periodic() {
+    // This method will be called once per scheduler run
+    SmartDashboard.putNumber("hood/Velocity DPS", getHoodVelocity());
```
Comment (alicekuznetsov) [inline src/main/java/frc/robot/subsystems/Hood.java]: All of these keys should be Hood/  not hood/

**Review by alicekuznetsov — CHANGES_REQUESTED** [review]

**Comment by TaylerUva** [comment]
Too stale :(

**Pull request closed without merging**